picoquic/frames.c

```diff
@@ -620,17 +620,19 @@ uint8_t* picoquic_decode_retire_connection_id_frame(picoquic_cnx_t* cnx, uint8_t
         /* If there is no matching path, trigger an error */
         picoquic_connection_error(cnx, PICOQUIC_TRANSPORT_PROTOCOL_VIOLATION,
             picoquic_frame_type_retire_connection_id);
+        bytes = NULL;
     }
```

Comment:
I added that here since almost all other `picoquic_connection_error` calls also set bytes = NULL.

After I did that, the `parse_frame_test()` unittest broke because the sequence number check fails in that test.